spec/requests/post_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :request do
+  let(:user) { User.create(name: 'John Doe') }
+
+  describe 'GET /users/:user_id/posts' do
+    it 'returns a successful response' do
+      get user_posts_path(user)
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the index template' do
+      get user_posts_path(user)
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      get user_posts_path(user)
+      expect(response.body).to include('post index')
+    end
+  end
+end
```

Comment:
- [ ] Please kindly implement the missing test for the `post_show` controller as well in accordance to the project requirement. ℹ️ 

<img width="384" alt="Untitled" src="https://github.com/geekelo/rails-blog_app/assets/103386183/8f66dc80-5bf1-4fe8-9f0c-6418d4b113ea">

Reply:
@OyePriscilla Thank you
Kindly do a favor by giving me a star. Thanks in advance